internal/inputhandlers/login.go

```diff
@@ -155,7 +155,7 @@ func LoginInputHandler(clientInput *connections.ClientInput, sharedState map[str
 			})
 
 			newUserPromptPrompt, _ := templates.Process("generic/prompt.yn", map[string]any{
-				"prompt":  "Would you like to create a new user?",
+				"prompt":  templates.T("Would you like to create a new user?"),
 				"options": []string{"y", "n"},
```

Comment:
Since these are used frequently, should we create a more universal `Yes` and `No`?

```
"options": []string{language.TextYes(), language.TextNo()},
```

This would also possibly standardize a lot of this, as we sometimes use "yes", "no", and other times "y", "n"
They can probably implement the Stringer interface for translation purposes, but remain directly comparable.

There are probably some other universal menu options worth considering:

`language.TextQuit()`
`language.TextSkip()`
`language.TextNext()`
`language.TextNew()`

By defining a pattern we could hopefully keep things more standardized, and is also a little easier to visually parse.

Reply:
Well, this can be considered, for the text display and command interaction will be a little different, just display direct translation is possible, but need to interact with the command may require more processing, we can decide how to deal with this in the future.